pom.xml

```diff
@@ -198,6 +200,78 @@
 
   <build>
     <plugins>
+      <plugin>
+        <groupId>org.apache.rat</groupId>
+        <artifactId>apache-rat-plugin</artifactId>
+        <version>0.12</version>
+        <executions>
+          <execution>
```

Comment:
Is this necessary? Should not `apache-rat:check` be already bound to the `validate` phase?

Reply:
This was here originally. I tried removing it. When it is removed the rat plugin is not executed when doing `mvn clean install`. So it seems to be required.

Reply:
I refer to `<phase>validate</phase>`.

Reply:
You are right, the docs say it is bound to the validate phase by default http://creadur.apache.org/rat/apache-rat-plugin/check-mojo.html . I also removed `<phase>validate</phase>` and tested that it work correctly.